zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplineServeTask.kt

```diff
@@ -61,7 +61,6 @@ abstract class ZiplineServeTask : DefaultTask() {
       val ws = websockets(
         "/ws" bind { ws: Websocket ->
           websockets.add(ws)
-          ws.send(WsMessage(RELOAD_MESSAGE))
```

Comment:
This is a deliberate change from the current implementation in Redwood which assumes that a reload is sent initially. Instead we immediately pass through and don't wait on websocket connect.